x/programs/runtime/program_invoke.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 const (
-	invokeModuleName = "program_invoke"
+	invokeModuleName = "program"
```

Comment:
why this change?

Reply:
in case we add more methods to the module